test/libsolidity/syntaxTests/literals/hex_string_misaligned_underscore.sol

```diff
@@ -0,0 +1,7 @@
+contract C {
+    function f() public pure {
+        hex"1_234";
+    }
+}
+// ----
+// ParserError: (52-56): Expected even number of hex-nibbles within double-quotes.
```

Comment:
This error message is misleading - the number of nibbles is even.

Reply:
Better now? It wasn't triggered on missing quotes anyways... and that part was actually even wrong, since single quotes are valid as well...